### PR TITLE
fix: replace Object.fromentries() to avoid version compatiblity issue

### PR DESCRIPTION
**Pull request by duidae**
twreporter-react升級node到16除了要改circle ci config外應該還要改其他地方, 而且最好需要測試, 而且url.parse那個warning改掉後routing會有問題, 還需要其他處理, 先把Object.fromentries替換成相容的寫法

### CODE REVIEW

File: packages/core/src/constants/category-set.js [hunk 1]
```diff
@@ -55,9 +55,10 @@ export const CATEGORY_ID = {
...
 
 // flip CATEGORY_ID to a {id: path} object
-const flippedCategoryID = Object.fromEntries(
-  Object.entries(CATEGORY_ID).map(([key, value]) => [value, key])
-)
+let flippedCategoryID = {}
+Object.entries(CATEGORY_ID).forEach(([key, value]) => {
+  flippedCategoryID[value] = key
+})
 export const GET_CATEGORY_PATH_FROM_ID = id => {
```
Comment (Aylie-Chou) [inline packages/core/src/constants/category-set.js:62]: How about using [`findKey`](https://lodash.com/docs/4.17.15#findKey) function provided by lodash here ?
```
export const GET_CATEGORY_PATH_FROM_ID = id => _.findKey(CATEGORY_ID,  (categoryId) => categoryId === id)

Comment (duidae) [reply packages/core/src/constants/category-set.js:62]: this is a very simple operation, not sure what would be the benefit to introduce a library here?

Comment (Aylie-Chou) [reply packages/core/src/constants/category-set.js:62]: IMO there's two benefits:
1. we don't need to store `flippedCategoryID` object.
2. current `core` package has `lodash` depency, we would not need to add other library for it.

Comment (Aylie-Chou) [reply packages/core/src/constants/category-set.js:62]: I think the trade off would be accepted.
Just one more thing here, the naming `flippedCategoryID` could not be easily understand that its key is `path`, which makes L63 a bit confusing.
Could we change the naming to contain some path-to-id meaning for it ?

Comment (duidae) [reply packages/core/src/constants/category-set.js:62]: renamed, please have a look, thanks.

**Review by Aylie-Chou — APPROVED** [review]
LGTM

**Pull request merged**